js/packages/components/chat/ContactSettings.tsx

```diff
@@ -35,7 +35,10 @@ const ContactSettingsHeader: React.FC<{ contact: any }> = ({ contact }) => {
 
 	const persistOpts = usePersistentOptions()
 	const isBetabot =
-		persistOpts && contact.publicKey.toString() === persistOpts.betabot.convPk.toString()
+		persistOpts &&
+		persistOpts.betabot &&
+		persistOpts.betabot.convPk &&
```

Comment:
This can be refracto into `persistOpts?.betabot?.convPk`

Reply:
I am setting a default value to `persistOpts.betabot.convPk` (and the other entries) now